jenkins/automation/reports/repo-health.groovy

```diff
@@ -0,0 +1,57 @@
+pipeline {
+    agent {
+        node {
+            label 'docker-k8-deployment-node'
+        }
+    }
+
+    options {
+        timeout(time: 420, unit: 'MINUTES')
+        timestamps()
+    }
+    
+    environment {
+        GH_OATH = credentials('shailesh-github-token')
```

Comment:
change `OATH` to `AUTH` everywhere in groovy and shell script

Reply:
This is a requirement from https://github.com/Seagate/cortx/blob/main/metrics/Repo_Health.ipynb

Reply:
Okay